manage.py

```diff
@@ -35,5 +35,24 @@ def test():
     unittest.TextTestRunner(verbosity=2).run(tests)
 
 
+@manager.command
+def setup_dev():
```

Comment:
That's a very good point. Agree that it's okay for setup_dev and setup_prod to be the same for now (it is better foundation for future).